Sources/PackageDescription4/Package.swift

```diff
@@ -43,72 +47,80 @@ import Foundation
 ///         ]
 ///     )
 ///
-/// # About the Swift Tools Version
-///
-/// A Package.swift manifest file must begin with the string `//
-/// swift-tools-version:` followed by a version number specifier.
-///
-/// Examples:
+/// A `Package.swift` manifest file must begin with the string `//
+/// swift-tools-version:` followed by a version number specifier. The following code listing shows a few examples of valid declarations of the Swift tools version:
 ///
 ///     // swift-tools-version:3.0.2
 ///     // swift-tools-version:3.1
 ///     // swift-tools-version:4.0
 ///     // swift-tools-version:5.0
 ///
 /// The Swift tools version declares the version of the `PackageDescription`
-/// library, the minimum version of the Swift tools and Swift language
+/// framework, the minimum version of the Swift tools and Swift language
```

Comment:
PackageDescription is not technically a "framework" so let's just call it a library.